.buildkite/pipeline.yml

```diff
@@ -56,9 +56,9 @@ steps:
       # call with bash explicitly because we run this on Windows, too
       bash .buildkite/commands/install-node-dependencies.sh
       echo '--- :package: Package app for testing'
-      npm run package
+      IS_DEV_BUILD=true npm run package
       echo '--- :playwright: Run End To End Tests'
-      npm run e2e
+      IS_DEV_BUILD=true npm run e2e
```

Comment:
It should be possible to DRY this by declaring this env var in the `env` node below.

```yml
env:
  IS_DEV_BUILD: true
```

See also:

- https://buildkite.com/docs/pipelines/configure/environment-variables#environment-variable-precedence-job-environment
- https://buildkite.com/docs/pipelines/configure/step-types/command-step#:~:text=env,Example%3A%20RAILS_ENV%3A%20%22test%22

Reply:
Thanks for the review!
I tried to add it to the `env` variable, but it still triggered the Sentry release (which means it didn't get passed onto the `npm` commands). [For this commit](https://github.com/Automattic/studio/pull/813/commits/9138adf677608df0bccb3ee7cecc490ba7045345)
On Sentry we can see:
![image](https://github.com/user-attachments/assets/84d8982d-1c0f-48f0-8cb7-04c7e23937a8)

I still tried to not repeat the `IS_DEV_BUILD` variable so much, and did `export IS_DEV_BUILD=true` before running `npm` in the `command` for each of the required steps.
Let me know what you think.

Reply:
Hey @bcotrim 

> I tried to add it to the `env` variable, but it still triggered the Sentry release (which means it didn't get passed onto the npm commands). [For this commit](https://github.com/Automattic/studio/pull/813/commits/9138adf677608df0bccb3ee7cecc490ba7045345)

Oh, that's odd. I'd love to investigate (knowing Buildkite works the way I expect it to is important for my day-to-day) but I don't have the time right now. 

Besides, your DRY solution via `export` does the job nicely.